judo-ui-react/src/main/resources/actor/src/utilities/filter-helper.ts.hbs

```diff
@@ -14,6 +14,18 @@ type FilterBy = {
   operator: Operation;
 };
 
+export enum _FilterOperationType {
+  EQUAL = 'equal',
```

Comment:
What about `equals`? :)

Reply:
This is the modelled filter operation types :)